### DISCUSSION TITLE
🔥Cura Community ~ Fire Side Chat🔥

### DISCUSSION THREAD

**Discussion by MariMakes** [Ultimaker/Cura#15657] [Collaboration]
![Untitled design (27)](https://github.com/Ultimaker/Cura/assets/40423138/c3e7d6f5-00b4-4f2c-b365-fdd308aa40f8)

We want to start a new tradition. At the end of every month, we want to take a moment to listen to what our moderators and community members want to share. We can also share a few improvements that we are expecting and ways we want to share our appreciation for your contributions. 💕 

Do you want to join our online call? 
Please fill in the form below to receive a zoom invite.
### [Click here to sign-up](https://forms.gle/d95yJc9QxroT2ZJA9)

Hopefully, we can find a time that works for everyone, or we'll split up to fit everyone's needs. 🤞 
If you are not available for the upcoming one, don't worry we plan to have one every month and we'll send you the notes. 💪
Feel free to tag anyone that you think would like to receive this invite and join us for a chat. 👍 

## Dec 6th, UTC [13:00  ](https://dateful.com/convert/utc?t=13&d=2024-12-06)

**Comment by MariMakes** [comment]
I promised the end of June, unfortunately, that promise was a bit too ambitious with the work around the Cura 5.4 release. 😰 I apologize for the delay.
### But we have a date now! 🎉 


```
Cura Community Fireside Chat Number 1
Next week: Friday, July 7th 12:00 UTC
```

If you want to join, please fill in the form above to sign up. 
If you cannot make it this time, no big deal, we want to make this a monthly thing.

**Reply by fieldOfView** [reply]
Hah, thanks for the delay! I knew I had seen the invitation come by somewhere, but for the life of me I could not find it back.
I'm in!

**Comment by GregValiant** [comment]
Do I need to install any particular software?

**Reply by MariMakes** [reply]
Not that I know off 🤔 I believe Zoom prompts the installation itself once you start the meeting if you don't have the software available.

**Comment by GregValiant** [comment]
I'm sitting here.

**Reply by MariMakes** [reply]
Can you check the link in your inbox again? We don't see you in the call

**Comment by GregValiant** [comment]
I didn't get the link.
Greg

**Comment by MariMakes** [comment]
Next week Friday will be the next Cura Community Fire Side Chat. 

Would you be interested in joining @printingotb, @Piezoid, @Cycov?
If you complete the form above I can send you an invite.

**Reply by MariMakes** [reply]
🤛 Bump! We have another one coming up this Friday, would you be interested to join?
[If you sign up I can send you the invite ](https://forms.gle/d95yJc9QxroT2ZJA9)

**Comment by giaco-leopardi** [comment]
Hello everyone
I was wondering if there is an option to customize the orbit settings to be the same as in Fusion 360 or Solidworks . I think it would be great if there is an option to change them by unifying the different softwares that a creator uses in its arsenal  and make the transition between the different softwares and the various stages of the development of a prototype more seamless.

**Comment by jellespijker** [comment]
@TheSin- is this something that you might be interested in?

**Reply by TheSin-** [reply]
The Fire Side Chat? Yes I am, I'm just dealing with a work crisis ATM, but I wanted to check dates and times to make sure I could make it before committing.

**Reply by jellespijker** [reply]
No rush or pressure, this is a monthly recurring thing.

**Reply by MariMakes** [reply]
Hey @TheSin- 

Good luck with your work crisis. It sounds important 🏃‍♂️ 
The upcoming one is September 29th, UTC 12:00.
[If you sign up I can send you the invite](https://forms.gle/d95yJc9QxroT2ZJA9)

**Reply by TheSin-** [reply]
Thank you.  I'm going to try to join in 12 UTC is early for me, but not unmanageable.

**Comment by qba1990** [comment]
Hello Cura community,

Sorry if I'm asking in a wrong place but I'm struggling with oozing problem using large nozzles. Retraction Extra Prime Amount is the setting that can help with that issue but it is "dummy" and its usage fix issues on longer travels but cause overextrusion on shorter ones. There was plugin that seems to resolve that problem - https://github.com/Pheneeny/CuraPlugin-ScalableExtraPrime. Prime in that plugin was being calculated basing on travel distance. Unfortunately that project is not being maintaned for a long time. Do You now if there is any alternative for that solution in current Cura? If not is here anyone who could help me with updating that plugin to be compatible with current Cura version? Any help would be much appreciated. Thank You and once again sorry if that is wrong place for such questions.

**Reply by qba1990** [reply]
To be precise - problem is not with oozing itself because it is consequence of using large nozzle and long travel but that oozing causes big gaps on seam (part of filament is leaking from hot end during travel and we need to refill that part using extra prime). That's why I need extra prime functionality but computed basing on travel distance or travel time.

**Reply by 5axes** [reply]
Not the right place for this discussion go to : https://github.com/Ultimaker/Cura/issues/12670

... But an alternative solution : https://github.com/5axes/CuraPlugin-ScalableExtraPrime (Last update June 2022)

**Reply by qba1990** [reply]
Thank You

**Comment by mikeydoom** [comment]
I'm glad to finally an Ender 5 S1 profile in the new beta.

However there are a couple of issues wrong with the profile.

The Z height is set to 305.0, it should be 280.0

The default retraction distance is 2mm, it should be 0.8.

That's all that's wrong.

**Comment by GregValiant** [comment]
@thomaspleasance put the files together for that.  If you would contact him he may have an explanation of why he added the setting values that he used.

**Comment by ox3673** [comment]
Please share this in an appropriate place if you would as I am not finding this sight overly user friendly for a none cpu person like me. either way, I think all slicers should have a machine usage even wear option: I like to move my prints around to even timing belt, rail, wheels and platform wear. I mean creating a random placement would probably be super easy for one of you programmers but creating a even use function might be more hard: say breaking the x,z axis travel into 1 inch blocks and logging use and runtime and placing models to even machine wear- it would be a useful function that I am surprised doesn't already exist. Also I have a K1 Max and I notice there is no listing, I have been using it and I think cura is the best or the one I like and hope a K1 max option will soon be available as creality print doesn't even load on my laptop for some odd reason. Hope it is a good idea or helps people.

**Reply by fieldOfView** [reply]
The Mesh Tools plugin has an option to randomize model placement on load. It is available for free in the Marketplace.

https://github.com/fieldOfView/Cura-MeshTools/#randomise-location

**Comment by jellespijker** [comment]
@chipmunk1977 would you like to join the next fireside chat.

**Reply by chipmunk1977** [reply]
I wasn't there last month, but normally I take part.

**Comment by TheSin-** [comment]
@MariMakes I'm sorry I missed the last FireSide, I tried to email you but I don't think I have to right email.  I'd really like to be part of the Nov FireSide.

**Comment by GregValiant** [comment]
12-1 at 8:00 AM my time.  Got it.
I have the analog switch on my laptop camera figured out.  It slides to the right to uncover the lens.
Pathetic really.
_"Getting old ain't for sissies."_ - Phyllis Diller

**Reply by MariMakes** [reply]
Yay! See you there 😄

**Comment by zBeeble42** [comment]
I can't start a new chat.  I created a new post on the Cura forums.   They pointed me back here.

So... I'd like to have Cura running on FreeBSD.  Cura 4.x built on FreeBSD.  So... I started poking at it.

The build for CuraEngine is pretty complex, but the issue that's sticking is that some portion of the build is putting -std=gnu++11 on the command line of the compiler but somewhere in protobuf requires C++14 as a minimum standard.  It says so with a static_assert().

I've gotten the build system to add either -std=c++14 or -std=gnu++14, but the -std=gnu++11 remains.  Very disheartening.

My latest effort's build log is here: https://v4.nextcloud.towernet.ca/s/3MM5cKxSjD6NzZ4 ... and I'd love to have my own discussion for this :).

**Comment by Kplay97** [comment]
Where can I get help???  This whole app is confusing and i don't understand it

**Comment by Kplay97** [comment]
I can't find any layer lines in my gcode notes.  Zero z instructions in the whole thing

**Comment by GregValiant** [comment]
This page is a discussion group for the moderators, contributors and such to pass notes for the monthly meetings.  If you require support or need questions answered you should post on Reddit in the Cura sub-reddit or on the UM Forum.  Reddit also has sub-reddits for particular printer models.
Be concise.  Consider that you never mentioned what printer you are having a problem with or what operating system you have or what Cura version you are trying to use.
After installing Cura to your system and then your printer in Cura, hit the blue "Slice" button.

**Comment by Kplay97** [comment]
Thank you 


Sent from Yahoo Mail for iPhone


On Sunday, March 10, 2024, 7:16 PM, GregValiant ***@***.***> wrote:




This page is a discussion group for the moderators, contributors and such to pass notes for the monthly meetings. If you require support or need questions answered you should post on Reddit in the Cura sub-reddit or on the UM Forum. Reddit also has sub-reddits for particular printer models.
Be concise. Consider that you never mentioned what printer you are having a problem with or what operating system you have or what Cura version you are trying to use.
After installing Cura to your system and then your printer in Cura, hit the blue "Slice" button.

**Comment by flavio170385** [comment]
buongiorno, vorrei segnalare un errore sulle impostazioni di una stampante. non so come contattare gli autori del programma quindi lo scrivo qui nella speranza che qualcuno possa aiutarmi. se seleziono la stampante Creality ender 3 v3 ke nelle impostazioni di velocità arriva a 150 mm/s anzichè 500mm/s. non so come cambiare personalmente il g code quindi chiedo un aiuto. grazie

**Comment by GregValiant** [comment]
There is an Italian sub-forum within the main [Ultimaker Forum](https://community.ultimaker.com/forum/127-italiano/).  You can ask for help in either one.
If you are using Creality Cura then you must ask over on the Creality support site.

In the printer definition file the Maximum Speed of the X and Y axis is 500mm/sec.  That is very likely too high to print or travel at with your printer.

If you are trying to adjust the speed from the LCD then that is "Percentage" and not speed.  If you were to slice a model using a print speed of 60mm/sec, and set the LCD to 150%, then your effective speed would be 90mm/sec.

**Comment by Asterchades** [comment]
Y'know it has just dawned on me that your system still doesn't want to automatically send me the invitations to this. Not that I feel like my contributions will be all that meaningful (my mind has been scattered enough that I've not noticed it's been *months* since last I attended) but I still feel bad for missing them.

Any solution you can think of beyond looking for a new email provider?

**Comment by TheSin-** [comment]
I did not get an email for 2025 yet, is there none planed anymore?

**Reply by Slashee-the-Cow** [reply]
Good to know I'm not the only one.

**Reply by TheSin-** [reply]
I talked with @MariMakes and she told me they are trying to get organized for March.
